include/picongpu/plugins/ResourceLog.hpp

```diff
@@ -60,7 +60,7 @@
 namespace boost_swap_impl
 {
     BOOST_GPU_ENABLED
-    void swap(std::string& s1, std::string& s2)
+    inline void swap(std::string& s1, std::string& s2)
```

Comment:
Why not `HINLINE` here?

Reply:
As I understood from the comment, `BOOST_GPU_ENABLED` can already be `__host__ __device__` (depending on build options) so just adding pure `inline`.